openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -190,6 +240,16 @@
                     view.rescheduleUnfinishedTasks();
                 }
             );
+
+            // Install a click handler for showing the staff grading form.
+            $staffGradeTool.find('.staff__grade__control').click(
+                function(eventObject) {
+                    // Don't call preventDefault because this click handler will be triggerred by
+                    // things like the radio buttons within the form-- we want them to be able to
```

Comment:
@andy-armstrong I find this rather ugly (the fact that we go in loadStaffGradeForm for all click events which happen within the form)-- curious if you have a suggestion.